requirements/base.in

```diff
@@ -1,10 +1,12 @@
 # Core requirements for using this application
 
-celery >= 3.1,<4.0
-Django                    # Web application framework
+-c constraints.txt
+
 # Why a django-celery fork? To add Django 1.11 compatibility to the abandoned django-celery repo.
 # This dependency will be removed by the Celery 4 upgrade: https://openedx.atlassian.net/browse/PLAT-1684
 -e git+https://github.com/edx/django-celery.git@f87c6f914a1410463f54aebf68458c0653b20602#egg=django-celery==3.2.1+edx.1
```

Comment:
Nit: I understand that the primary purpose of this PR is a security update. But it would be nice to attempt to get us back on the upstream of django-celery at the same time. I checked and the upstream version *is* compatible with Django 1.11 now.

Reply:
Thanks for the review @doctoryes! @DawoudSheraz could you try to upgrade this as well? I think it'd be a small thing that would help with the sustainability of our system.

Reply:
@fysheets @doctoryes Sure. I will try the latest version and see if there are any test failures.